src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationParsingTests.cs

```diff
@@ -7319,15 +7314,10 @@ public void TestNullCheckedArgList()
         }
 
 
-        [Fact(Skip = "PROTOTYPE(BangBang)")]
+        [Fact]
         public void TestNullCheckedArgWithLeadingSpace()
```

Comment:
>TestNullCheckedArgWithLeadingSpace [](start = 20, length = 34)

Are the tokens in the following three tests distinct? If not, please consider merging into a single test method using `[Theory][InlineData("...")]...".